### ISSUE TITLE
Does not work in Ruby 1.9.1 but does not declare this requirement

### ISSUE THREAD

**Issue by wtanaka** [coderanger/kitchen-sync#23]
https://travis-ci.org/wtanaka/role-tester-ansible/jobs/143697669#L291

```
-----> Starting Kitchen (v1.10.2)
/home/travis/build/wtanaka/role-tester-ansible/vendor/bundle/ruby/1.9.1/gems/test-kitchen-1.10.2/lib/kitchen/transport.rb:40:in `require': /home/travis/build/wtanaka/role-tester-ansible/vendor/bundle/ruby/1.9.1/gems/kitchen-sync-2.1.1/lib/kitchen/transport/sftp.rb:89: syntax error, unexpected tLABEL (SyntaxError)
        def sftp_upload!(local, remote, recursive: true, purge: true)
                                                  ^
/home/travis/build/wtanaka/role-tester-ansible/vendor/bundle/ruby/1.9.1/gems/kitchen-sync-2.1.1/lib/kitchen/transport/sftp.rb:89: Can't assign to true
        def sftp_upload!(local, remote, recursive: true, purge: true)
                                                        ^
/home/travis/build/wtanaka/role-tester-ansible/vendor/bundle/ruby/1.9.1/gems/kitchen-sync-2.1.1/lib/kitchen/transport/sftp.rb:243: syntax error, unexpected keyword_end, expecting $end
	from /home/travis/build/wtanaka/role-tester-ansible/vendor/bundle/ruby/1.9.1/gems/test-kitchen-1.10.2/lib/kitchen/transport.rb:40:in `for_plugin'
	from /home/travis/build/wtanaka/role-tester-ansible/vendor/bundle/ruby/1.9.1/gems/test-kitchen-1.10.2/lib/kitchen/config.rb:313:in `new_transport'
	from /home/travis/build/wtanaka/role-tester-ansible/vendor/bundle/ruby/1.9.1/gems/test-kitchen-1.10.2/lib/kitchen/config.rb:253:in `new_instance'
	from /home/travis/build/wtanaka/role-tester-ansible/vendor/bundle/ruby/1.9.1/gems/test-kitchen-1.10.2/lib/kitchen/config.rb:144:in `block in build_instances'
	from /home/travis/build/wtanaka/role-tester-ansible/vendor/bundle/ruby/1.9.1/gems/test-kitchen-1.10.2/lib/kitchen/config.rb:143:in `map'
	from /home/travis/build/wtanaka/role-tester-ansible/vendor/bundle/ruby/1.9.1/gems/test-kitchen-1.10.2/lib/kitchen/config.rb:143:in `with_index'
	from /home/travis/build/wtanaka/role-tester-ansible/vendor/bundle/ruby/1.9.1/gems/test-kitchen-1.10.2/lib/kitchen/config.rb:143:in `build_instances'
	from /home/travis/build/wtanaka/role-tester-ansible/vendor/bundle/ruby/1.9.1/gems/test-kitchen-1.10.2/lib/kitchen/config.rb:119:in `instances'
	from /home/travis/build/wtanaka/role-tester-ansible/vendor/bundle/ruby/1.9.1/gems/test-kitchen-1.10.2/lib/kitchen/command.rb:97:in `all_instances'
	from /home/travis/build/wtanaka/role-tester-ansible/vendor/bundle/ruby/1.9.1/gems/test-kitchen-1.10.2/lib/kitchen/command.rb:145:in `parse_subcommand'
	from /home/travis/build/wtanaka/role-tester-ansible/vendor/bundle/ruby/1.9.1/gems/test-kitchen-1.10.2/lib/kitchen/command/test.rb:43:in `block in call'
	from /usr/local/rvm/rubies/ruby-1.9.3-p551/lib/ruby/1.9.1/benchmark.rb:280:in `measure'
	from /home/travis/build/wtanaka/role-tester-ansible/vendor/bundle/ruby/1.9.1/gems/test-kitchen-1.10.2/lib/kitchen/command/test.rb:41:in `call'
	from /home/travis/build/wtanaka/role-tester-ansible/vendor/bundle/ruby/1.9.1/gems/test-kitchen-1.10.2/lib/kitchen/cli.rb:56:in `perform'
	from /home/travis/build/wtanaka/role-tester-ansible/vendor/bundle/ruby/1.9.1/gems/test-kitchen-1.10.2/lib/kitchen/cli.rb:231:in `test'
	from /home/travis/build/wtanaka/role-tester-ansible/vendor/bundle/ruby/1.9.1/gems/thor-0.19.1/lib/thor/command.rb:27:in `run'
	from /home/travis/build/wtanaka/role-tester-ansible/vendor/bundle/ruby/1.9.1/gems/thor-0.19.1/lib/thor/invocation.rb:126:in `invoke_command'
	from /home/travis/build/wtanaka/role-tester-ansible/vendor/bundle/ruby/1.9.1/gems/test-kitchen-1.10.2/lib/kitchen/cli.rb:325:in `invoke_task'
	from /home/travis/build/wtanaka/role-tester-ansible/vendor/bundle/ruby/1.9.1/gems/thor-0.19.1/lib/thor.rb:359:in `dispatch'
	from /home/travis/build/wtanaka/role-tester-ansible/vendor/bundle/ruby/1.9.1/gems/thor-0.19.1/lib/thor/base.rb:440:in `start'
	from /home/travis/build/wtanaka/role-tester-ansible/vendor/bundle/ruby/1.9.1/gems/test-kitchen-1.10.2/bin/kitchen:13:in `block in <top (required)>'
	from /home/travis/build/wtanaka/role-tester-ansible/vendor/bundle/ruby/1.9.1/gems/test-kitchen-1.10.2/lib/kitchen/errors.rb:174:in `with_friendly_errors'
	from /home/travis/build/wtanaka/role-tester-ansible/vendor/bundle/ruby/1.9.1/gems/test-kitchen-1.10.2/bin/kitchen:13:in `<top (required)>'
	from /home/travis/build/wtanaka/role-tester-ansible/vendor/bundle/ruby/1.9.1/bin/kitchen:23:in `load'
	from /home/travis/build/wtanaka/role-tester-ansible/vendor/bundle/ruby/1.9.1/bin/kitchen:23:in `<main>'
make: *** [test] Error 1
The command "./_self_test.sh" exited with 2.
```

**Comment by wtanaka** [comment]
2.1.0 seems to be working.

https://travis-ci.org/wtanaka/role-tester-ansible/jobs/143746269

**Comment by coderanger** [comment]
In general I only support whatever version of Ruby is being used for Chef's installers which I think is currently 2.1. Ruby 1.9 (and 2.0 for that matter) are EOL and no longer supported by the Ruby team. As such few gems support them anymore.

**Comment by wtanaka** [comment]
If the rubygems meta data reflected that, this would fail at bundler time with a much clearer error.
 
 
  On Sun, Jul 10, 2016 at 9:15, Noah Kantrowitz<notifications@github.com> wrote:   
Closed #23.
